### PR TITLE
Get units values

### PR DESCRIPTION
**Pull request by phlax**
Adds classes for parsing Unit.values_lists from get_units searches

### CODE REVIEW

File: pootle/apps/pootle_store/unit/results.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    @property
+    def target_lang(self):
+        return self.unit[
+            "store__translation_project__language__code"]
+
+    @property
+    def translate_url(self):
+        store_url = u''.join(
+            [reverse("pootle-tp-store-translate",
+                     args=split_pootle_path(self.pootle_path))])
```
Comment (unho) [inline pootle/apps/pootle_store/unit/results.py]: Nit: Why not just `str()` it?

Comment (phlax) [inline pootle/apps/pootle_store/unit/results.py]: agreed - i coped out this code - ill update

File: pootle/apps/pootle_store/unit/results.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        self.units = units
+
+    @property
+    def data(self):
+        meta = None
+        units_list = []
+
+        for unit in iter(self.units):
+            unit = UnitResult(unit)
+            if meta is None:
```
Comment (unho) [inline pootle/apps/pootle_store/unit/results.py]: Doing this for all units looks like a bit inefficient, so better do it once before `for`.

**Comment by unho** [comment]
lgtm once nits addressed.

File: pootle/apps/pootle_store/unit/results.py (hunk 2 above)
Comment (phlax) [inline pootle/apps/pootle_store/unit/results.py]: no - at least in the current implementation it needs the unit.

File: pootle/apps/pootle_store/views.py [hunk 3]
```diff
@@ -240,38 +241,6 @@ def _prepare_unit(unit):
...
-            meta = {
-                'source_lang': project.source_language.code,
-                'source_dir': project.source_language.direction,
-                'target_lang': tp.language.code,
-                'target_dir': tp.language.direction,
-                'project_code': project.code,
-                'project_style': project.checkstyle,
-            }
-
-        units_list.append(_prepare_unit(unit))
```
Comment (ta2-1) [inline pootle/apps/pootle_store/views.py]: @phlax, will we get rid of _prepare_unit?

Comment (phlax) [inline pootle/apps/pootle_store/views.py]: i think so - but its still used elsewhere atm

**Comment by ta2-1** [comment]
lgtm

**Pull request merged**